app/views/lists/show.html.haml

```diff
@@ -0,0 +1,19 @@
+%h5.float-center= @list.name
+- books = []
+- books_id = []
+- @books_ids.each{ |books| books_id.push(books.book_id) }
+- books_id.each do |book|
+  - books.push(Book.find(book))
```

Comment:
what a hell with this ruby code in view? some weird functionality, random use of `books_ids` and `@books_ids`? what?